src/components/index.js

```diff
@@ -0,0 +1,6 @@
+export { default as Toggle } from './Toggle' 
+export { default as SearchForm } from './SearchForm' 
+export { default as Result } from './Result' 
+export { default as HelmetStyles } from './HelmetStyles' 
+export { default as Avatar } from './Avatar' 
+export { default as Nav } from './Nav' 
```

Comment:
I'm sure you've done this elsewhere, but great use of destructured importing/exporting